docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Density
+
+You can change the density of the rows and the column header.
+
+### Size prop
+
+There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the `DataGrid` is set to `medium`. The default grid density can be changed by applying the `size` prop. The `size` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
+
+{{"demo": "pages/components/data-grid/rendering/DensityPickerSmallGrid.js", "bg": "inline"}}
+
+### Density picker button
+
+The default way to change the density of the data grid is by using the `DensityPicker` component from the toolbar. In addition, there is a prop to hide the `DensityPicker`.
```

Comment:
This component is private, developers shouldn't know about it.

```suggestion
The default way to change the density of the data grid is by using the density picker component from the toolbar. In addition, there is a prop to hide the density picker.
```

Reply:
```suggestion
The default way for a user to change the density of the data grid is by using the density picker component from the toolbar. In addition, there is a prop to hide the density picker.
```